src/olympia/promoted/models.py

```diff
@@ -92,7 +92,7 @@ def has_pending_subscription(self):
         return (
             self.group.require_subscription and
             (subscr := getattr(self, 'promotedsubscription', None)) and
-            not subscr.stripe_checkout_completed and
+            not subscr.is_active and
```

Comment:
can you add something to `TestPromotedAddon.test_has_pending_subscription` that covers the subscription being cancelled?  (or change the asserts on `stripe_checkout_completed` to `is_active`)